library/core/src/str/mod.rs

```diff
@@ -4076,11 +4076,12 @@ impl str {
 
     /// Returns a string slice with the prefix removed.
     ///
-    /// If the string starts with the pattern `prefix`, `Some` is returned with the substring where
-    /// the prefix is removed. Unlike `trim_start_matches`, this method removes the prefix exactly
-    /// once.
+    /// If the string starts with the pattern `prefix`, returns
```

Comment:
these lines are wrapped much, much shorter than the old ones were. What column did you wrap these to?